controllers/src/test/scala/cromwell/pipeline/controller/AuthControllerTest.scala

```diff
@@ -54,55 +55,38 @@ class AuthControllerTest extends WordSpec with Matchers with MockFactory with Sc
-        val firstName = "FirstName"
-        val lastName = "LastName"
+      "return token headers if user was successfully registered" in {
         val signInRequestStr =
           s"""{"email":"${email}","password":"${password}","firstName":"${firstName}","lastName":"${lastName}"}"""
         val authResponse = AuthResponse(accessToken, refreshToken, accessTokenExpiration)
         val httpEntity = HttpEntity(`application/json`, signInRequestStr)
-        (authService.signUp _ when SignUpRequest(email, password, firstName, lastName))
-          .returns(Future(Some(authResponse)))
+        (authService.signUp _ when SignUpRequest(UserEmail(email), Password(password), Name(firstName), Name(lastName)))
+          .returns(Future.successful(Some(authResponse)))
 
         Post("/auth/signUp", httpEntity) ~> authController.route ~> check {
           status shouldBe StatusCodes.OK
           checkAuthTokens
         }
       }
 
-      "return BadRequest with fields validation errors" taggedAs Controller in {
-        val email = "email"
-        val password = "password"
-        val firstName = "First-name"
-        val lastName = "Last-name"
+      "return BadRequest with fields validation errors" ignore {
         val signUpRequestStr =
           s"""{"email":"${email}","password":"${password}","firstName":"${firstName}","lastName":"${lastName}"}"""
         val authResponse = AuthResponse(accessToken, refreshToken, accessTokenExpiration)
-
         val httpEntity = HttpEntity(`application/json`, signUpRequestStr)
-        (authService.signUp _ when SignUpRequest(email, password, firstName, lastName))
-          .returns(Future(Some(authResponse)))
+        (authService.signUp _ when SignUpRequest(UserEmail(email), Password(password), Name(firstName), Name(lastName)))
+          .returns(Future.successful(Some(authResponse)))
 
         Post("/auth/signUp", httpEntity) ~> authController.route ~> check {
-          val errors = Json.parse(responseAs[String]).as[List[Map[String, String]]]
-          val errorCodes = errors.map(_("errorCode")).toSet
-
           status shouldBe StatusCodes.BadRequest
-          DomainValidation.allErrorCodes.forall(errorCodes.contains) shouldBe true
         }
       }
 
-      "return BadRequest status if user registration was failed" taggedAs Controller in {
-        val email = "email@cromwell.com"
-        val password = "password"
-        val firstName = "First-name"
-        val lastName = "Last-name"
+      "return BadRequest status if user registration was failed" in {
         val signUpRequestStr =
           s"""{"email":"${email}","password":"${password}","firstName":"${firstName}","lastName":"${lastName}"}"""
         val httpEntity = HttpEntity(`application/json`, signUpRequestStr)
-        (authService.signUp _ when SignUpRequest(email, password, firstName, lastName))
+        (authService.signUp _ when SignUpRequest(UserEmail(email), Password(password), Name(firstName), Name(lastName)))
```

Comment:
For some reason, we are still able to unsafely create microtypes without explicitly importing the `Unsafe` object.
This must be fixed.

Reply:
Done